### ISSUE TITLE
php-fpm:7.2-mods and pdo_sqlsrv extension

### ISSUE THREAD

**Issue by llaville** [devilbox/docker-php-fpm#197]
Since release 5.9.0 of pdo_sqlsrv, PHP 7.2 support was dropped (see https://pecl.php.net/package/pdo_sqlsrv/5.9.0).

Then we must fix pecl installation (in build/ansible/group_vars/all/mods.yml) such as follow : 

```
  pdo_sqlsrv:
    disabled: [5.2, 5.3, 5.4, 5.5, 5.6, 7.4, 8.0, 8.1]
    7.0:
      type: pecl
      version: 5.3.0
      build_dep: [unixodbc-dev]
      run_dep: [unixodbc]
    7.1:
      type: pecl
      version: 5.6.1
      build_dep: [unixodbc-dev]
      run_dep: [unixodbc]
    7.2:
      type: pecl
      version: 5.8.1
      build_dep: [unixodbc-dev]
      run_dep: [unixodbc]
    all:
      type: pecl
      build_dep: [unixodbc-dev]
      run_dep: [unixodbc]
```

**Comment by llaville** [comment]
We should also enable PHP 8.0 support (available since release 5.9.0)

```
  sqlsrv:
   disabled: [5.2, 5.3, 5.4, 5.5, 5.6, 7.4, 8.1]
```

**Comment by llaville** [comment]
As fixed is provide in alternative solution, I'll close this report. You may pick commit on my repository if you want !
